plugins/modules/ec2_spot_instance.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
```

Comment:
These two lines are specific to Python2. Since we now focus on Python3, you can just remove them.